packages/ia-components/sandbox/language-select/language-select.less

```diff
@@ -41,10 +41,15 @@
     &.visible {
       display: block;
     }
+    // It needs a more specific classname else the body.navia .navbar #nav-dweb ul { display:closed } beats the one below
```

Comment:
Is the actual property value you're overriding `display: block`? `closed` isn't a property value.

Reply:
Sorry its display:inline that it overrides, I've pushed an update